phpBB/includes/functions.php

```diff
@@ -921,6 +921,16 @@ function markread($mode, $forum_id = false, $topic_id = false, $post_time = 0, $
 
 		return;
 	}
+	else if ($mode == 'usermention')
```

Comment:
Yeah, Initially I did this but if you see the part in viewTopic.php where markread function is called, even if I leave the method call as it is and not define my own mode, I guess the " if " part is not executed, in fact the " else " part is executed, which has no markread function call.